internal/parser.go

```diff
@@ -59,7 +58,7 @@ func NewParser(cfg string, opts *Options, fs FileSystem) Parser {
 
 	// Clear cache if CLI flag was provided.
 	if opts.ClearCache && p.fs.FileExists(tempFile) {
-		p.fs.Remove(tempFile)
+		_ = p.fs.Remove(tempFile)
```

Comment:
I don't think this is necessary. What do you think?

Reply:
i usually do this to stop the linters from complaining that we're not handling errors